src/core/i18n/translator.js

```diff
@@ -0,0 +1,30 @@
+export default class Translator {
+  /**
+   * Performs a translation which supports
+   * interpolation, pluralization, or both
+   * @param {string} translations The translations, or a stringified JSON of possible translations
+   * @param {Object} interpolationParams Params to use during interpolation
+   * @param {number} count The count associated with the pluralization
+   */
+  static translate (translations, interpolationParams, count) {
+    const stringToInterpolate = this._selectPluralization(translations, count);
+    return this._interpolate(stringToInterpolate, interpolationParams);
+  }
+
+  static _selectPluralization (translations, count) {
```

Comment:
I had no idea you could use `this` to reference a static method

Reply:
Ya, it lets you call static functions defined below